internal/creds/aws.go

```diff
@@ -32,10 +30,13 @@ func ReadConfig(ctx context.Context, node *api.NodeConfig, opts ...func(*config.
 			awsConfigPath = iamrolesanywhere.DefaultAWSConfigPath
 		}
 
+		// we do not specify the iam-ra credentials file since this is used by the
+		// debug command which we want to match as close as possible to the kubelet which
+		// also does not use the iam-ra credentials file
```

Comment:
can we rename the method to something like `ReadConfigAsKubelet`? i want to prevent someone using this one when they should be using the other one